### PR TITLE
Newsweek-bootstrap: Milestone 3

### PR DESCRIPTION
**Pull request by felix-vondee**
In this milestone, we designed the final sections of the Newsweek homepage.

We tried to design the scroll effect that was seen on the original website, but it was quite advanced and required some javascript code. Our middle column ended up being longer than the other two columns on desktop view.

In our previous code review, the TSE informed us to cover the overflow that was present on the navbar which made some elements behind the navbar visible. However, on the original site, the thin navbar drops from the top when a certain scroll height is reached. We tried our best to replicate the navbar effect without javascript.

We also made some changes to our CSS file and used bootstrap classes in the HTML (as requested by the TSE).

Apart from that, we met all the requirements for milestone 3.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 🔴 ♻️ 

### Hey @felix-vondee & @Stephanie041996, 👋🏽 
**This is a great project you have going on, well-done lads. However, there are a few issues to fix to get approved.**

### REQUIRED CHANGES: ⚠️ 

- [x] In the magazine section, the text 'See all features' should be enclosed in an anchor tag. The same applies to the image above the text.
- [x] The `h4` containing the text 'NEWS', that's adjacent to the `h3` with text 'COVER' should be enclosed in an anchor tag.
- [x] The images in the Editor's pick should also be enclosed in anchor tags.
- [x] The Main newspaper sections (US, World, etc.) titles should be enclosed in anchor tags.

- [x] Your clone has this overflow once I start scrolling down the page. See what you can do to over that.
Screenshot_2021-03-03_15-02-21

- [x] The padding is a little off at the top for the third column. Check it out and adjust accordingly.
Screensh

- [x] Adjust the image sizes for mobile screens for this section below.
- Your clone
Screenshot_2021-03-03_15-24-57

- Original site
Screenshot_2021-03-03_15-26-23

- [x] Add the arrow inside the buttons, adjust the padding of the NewsLetter section to match the subscribe section.
- Your clone
Screenshot_2021-03-03_15-32-02

- [x] Refactor your CSS file. A number of CSS properties there can be added directly to the HTML structure. The CSS file should have 400 lines at most.



NOTE.

- [x] ALL THESE CHANGES CAN BE DONE WITHIN THE HTML FILE WITH BOOTSTRAP. 😄 

### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

## Reach me on slack for a chit-chat 🆘  Roy Mukuye

File: style.css [hunk 1]
```diff
@@ -0,0 +1,816 @@
...
+}
+
+img:hover {
+  cursor: pointer;
+}
+
+.top_nav > div > img {
+  width: 366px;
+  height: 85px;
+  cursor: pointer;
```
Comment (mke2111) [inline style.css:37]: Create a new class for `cursor: pointer;`. Then add this class to all the elements that need it in the HTML file. Let's keep the DRY rule respected.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,816 @@
...
+
+h4 > a {
+  font-weight: bold;
+  color: #a0a0a0;
+  font-size: 20px;
+}
+
+h6 > a {
+  font-weight: bold;
+  color: black;
```
Comment (mke2111) [inline style.css:65]: Create a new class for `color: black;` It has more than 9 appearaces. Then add this class to all the elements that need it in the HTML file. Let's keep the DRY rule respected.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,816 @@
...
+}
+
+h5 > a {
+  color: black;
+  font-weight: bold;
+  font-size: 17px;
+}
+
+.in_images > a {
+  margin-bottom: 0;
```
Comment (mke2111) [inline style.css:76]: Bootstrap has a class for this property. `mb-0`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,816 @@
...
+  padding: 5px 15px 6px;
+  background-color: rgba(255, 255, 255, 0.9);
+  font-weight: bold;
```
Comment (mke2111) [inline style.css:79]: Create a new class for `font-weight: bold;`. Then add this class to all the elements that need it in the HTML file. Let's keep the DRY rule respected.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,816 @@
...
+  text-decoration: none;
+  border-bottom: 1px solid red;
+}
+
+ul > li {
+  column-gap: 10px;
+}
+
+.nav_bar_mid {
+  width: 100%;
```
Comment (mke2111) [inline style.css:100]: Bootstrap has a class for this property

File: style.css [hunk 6]
```diff
@@ -0,0 +1,816 @@
...
+  height: 54px;
+}
+
+h4 > a:hover {
+  color: #a0a0a0;
+  text-decoration: none;
```
Comment (mke2111) [inline style.css:106]: Create a new class for `text-decoration: none;`. Then add this class to all the elements that need it in the HTML file. Let's keep the DRY rule respected.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,816 @@
...
+  color: black;
+  text-decoration: none;
+  border-bottom: 1px solid red;
+}
+
+.nav_bar_mid > li {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  width: 100%;
```
Comment (mke2111) [inline style.css:119]: Bootstrap has classes for display, justify content, margin and width

File: style.css [hunk 8]
```diff
@@ -0,0 +1,816 @@
...
+}
+
+.topstory {
+  row-gap: 15px;
+}
+
+.background1 {
+  background-image: url(https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1711616/donald-trump.webp?w=397&amp;h=265&amp;q=90&amp;f=d759057143d283aa0ea0ed13fba5450a);
+  background-size: cover;
+  background-position: center;
```
Comment (mke2111) [inline style.css:300]: Bootstrap has classes for these as well.

## Spare enough time and search about all classes you can use for this project. Minimize the number of CSS lines you have.

**Review by arslanbisharat — APPROVED** [review]
# Approved 🥇 

Thank you for submitting this project. You have done a great job!

- [x] You have a descriptive Pull Request 👏 
- [x] You have a descriptive Readme. 👏 
- [x] Your commit messages are very descriptive too. I am very happy to see that.  👏 
- [x] No Linter error found. 👏 
- [x] Met Minimum requirements to pass the project.  

I am very happy to review your project. Looking forward to amazing new projects from your side. 

**Best,
Arslan Bisharat**
_You can always contact me on slack at @arslanbisharat 📞_

**Pull request closed without merging**